### PR TITLE
DOC-2045: Updated node upgrade page for CE 4.6 to 4.8

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by gssoma — APPROVED** [review]
Looks good Anna

**Review by gssoma — APPROVED** [review]
Looks good Anna

**Review by gssoma — DISMISSED** [review]
Looks good Anna

File: content/en/docs/corda-enterprise/4.7/node-upgrade-notes.md [hunk 1]
```diff
@@ -215,22 +215,29 @@ For this step you can use any valid dummy name - for example, `O=Dummy,L=London,
...
 
-A script named `migrationYYYYMMDDHHMMSS.sql` will be generated in the current directory.
+A script named `migrationYYYYMMDDHHMMSS.sql` containing DDL and DML statements will be generated in the current directory.
 This script will contain all the statements required to modify and create data structures (for example, tables/indexes),
 and updates the Liquibase management table *DATABASECHANGELOG*.
 The command doesn’t alter any tables itself.
+
+{{< note >}}
+
+If your DDL and DML statements are run separately - DB admin runs DDL and an app user runs DML, you need to manually separate DDL and DML statements into separate scripts.
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.7/node-upgrade-notes.md:237]:
```suggestion
If your DDL and DML statements are run separately (for example, if the DB admin runs DDL and an app user runs DML) you need to manually separate the DDL and DML statements into separate scripts.
```

File: content/en/docs/corda-enterprise/4.7/node-upgrade-notes.md [hunk 2]
```diff
@@ -258,7 +265,13 @@ An accidental re-run of the scripts will fail (as the tables are already there),
 
 
 
-### 3.4. Apply data updates on a database
+### 3.4. Optional: Apply data updates on the H2 database
+
+{{< note >}}
+
+You do not need to perform this step for databases other than H2 database.
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.7/node-upgrade-notes.md:272]:
```suggestion
You only need to perform this step for the H2 database.
```

File: content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md [hunk 3]
```diff
@@ -215,22 +215,29 @@ For this step you can use any valid dummy name - for example, `O=Dummy,L=London,
 Copy the PostgreSQL JDBC Driver *42.2.8* version *JDBC 4.2* into the `drivers` directory.
 
 
-### 3.2. Extract DDL script using Database Management Tool
+### 3.2. Extract DDL and DML script using Database Management Tool
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md:218]:
```suggestion
### 3.2. Extract the DDL and DML scripts using Database Management Tool
```

File: content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md [hunk 4]
```diff
@@ -215,22 +215,29 @@ For this step you can use any valid dummy name - for example, `O=Dummy,L=London,
...
 
-A script named `migrationYYYYMMDDHHMMSS.sql` will be generated in the current directory.
+A script named `migrationYYYYMMDDHHMMSS.sql` containing DDL and DML statements will be generated in the current directory.
 This script will contain all the statements required to modify and create data structures (for example, tables/indexes),
 and updates the Liquibase management table *DATABASECHANGELOG*.
 The command doesn’t alter any tables itself.
+
+{{< note >}}
+
+If your DDL and DML statements are run separately - DB admin runs DDL and an app user runs DML, you need to manually separate DDL and DML statements into separate scripts.
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md:237]:
```suggestion
If your DDL and DML statements are run separately (for example, if the DB admin runs DDL and an app user runs DML) you need to manually separate the DDL and DML statements into separate scripts.
```

File: content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md [hunk 5]
```diff
@@ -258,7 +265,13 @@ An accidental re-run of the scripts will fail (as the tables are already there),
 
 
 
-### 3.4. Apply data updates on a database
+### 3.4. Optional: Apply data updates on the H2 database
+
+{{< note >}}
+
+You do not need to perform this step for databases other than H2 database.
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/node-upgrade-notes.md:272]:
```suggestion
You only need to perform this step for the H2 database.
```

**Review by erinking-r3 — APPROVED** [review]

**Pull request merged**